requirements.txt

```diff
@@ -1,10 +1,10 @@
-Pillow>=2.0.0
-coverage
-html5lib>=1.0
-pyPdf2==1.26
-reportlab>=3.0
-six
-Sphinx==1.4.8
-sphinx-rtd-theme==0.1.9
-python-bidi==0.4.2
 arabic-reshaper==2.1.0
+coverage==5.3
+html5lib==1.1
```

Comment:
html5lib>=1.1

Reply:
This part break some configurations, please try to use more >= than == because this help other projects to have his own version of packages.

Reply:
> This part break some configurations, please try to use more >= than == because this help other projects to have his own version of packages.

The requirements.txt file shouldn't bother others, that aren't helping to develop xhtml2pdf. It's just for the people that are forking the project and want to contribute and submit pull requests. And they should create a virtualenv anyway and then install the exact packages, to make everything reproducible.

And the "normal user" uses ``pip install xhtml2pdf``. Pip won't look at the ``requirements.txt`` file. It will only look at the ``setup.py`` file.

But please correct me, if I'm completely wrong. 😄 


Recently I read a good article, that summarizes this:
https://medium.com/knerd/best-practices-for-python-dependency-management-cc8d1913db82

Reply:
I know pip uses setup.py file, but for example github looks on requirements.txt,  developers also can try to run
` pip install -r requirements.txt`,  as I guest also read the docs install from requirements.txt.

So it's better to have a flexible requirements on `requirements.txt`

Reply:
If developers want to contribute and use ``pip install -r requirements.txt``, they should in my opinion create an extra venv for it.

I'm not too sure about Read-the-Docs. As far as I know, Travis is responsible for Sphinx and Read-the-Docs. In the ``.travis.yml`` file is says ``pip install tox coveralls Sphinx sphinx-rtd-theme``. But in the ``tox.ini`` file it says ``[testenv:develop]
deps = -rrequirements.txt`` So I'm not too sure.

But anyway: It's ok to have different opinions. :) I've changed all dependencies in ``requirements.txt`` to ``>=``.